### DISCUSSION TITLE
setting up the library for icon-component

### DISCUSSION THREAD

**Discussion by KheteshS** [MaibornWolff/tlm-components#219] [Q&A] [answered]
May I please know how you are setting up a library for the icon component: where are you storing the icons if not in assets....and after storing how is mapping done while retrieving the icon and using it as a word instead of giving the whole location of the icon file?

**Comment by Fuasmattn** [comment, marked as answer]
We're actually using web fonts for the icons and use a web component as a wrapper: https://fonts.google.com/icons
You can find more hints in the styling or the icon component
https://github.com/MaibornWolff/mwui/tree/main/mwui-stencil/src/components/mw-icon
